scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc

```diff
@@ -7,17 +8,25 @@ toc::[]
 
 include::modules/cnf-understanding-low-latency.adoc[leveloffset=+1]
 
+.Additional resources
+
+* xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#configuring_hyperthreading_for_a_cluster_{context}[Configuring hyperthreading for a cluster]
```

Comment:
I believe you can't use `{context}` in an xref, you must provide the actual value for the context variable _of the module you're referring to_ (right now, the context of this assembly is used, and the linked module has a different context). All underscores in the anchor ID also look suspicious.

Reply:
Same goes for both the other xrefs in the changeset.